### DISCUSSION TITLE
Slack access

### DISCUSSION THREAD

**Discussion by sdanyliv** [linq2db/linq2db#4042] [General]
@MaceWindu, something wrong with Slack? I cannot see `linq2db` Workplase.

**Comment by MaceWindu** [comment]
Should be this

![image](https://user-images.githubusercontent.com/1099238/226888026-97e64593-090c-4df4-ba7e-eb3762e734eb.png)

which is a dick move as we are not related to russia in any way. Igor as owner should contact them to claify things...

**Comment by sdanyliv** [comment]
Probably because of @ili. Can we do something with this?

**Reply by MaceWindu** [reply]
As I said, @igor-tkachev as slack workspace owner should contact them and try to resolve this situation, but I would recommend to start looking for more open-source friendly alternatives

**Reply by MaceWindu** [reply]
https://itsfoss.com/open-source-slack-alternative/

**Reply by sdanyliv** [reply]
I think @ili started Slack, not Igor.

**Reply by MaceWindu** [reply]
Ah, than it explains it.

**Reply by sdanyliv** [reply]
Try to write them something 

https://slack.com/blog/news/access-restricted-russia-embargoed-regions-ukraine

**Reply by MaceWindu** [reply]
I will check later today what we can use which supports github integration (azure integration will be a plus)

**Reply by sdanyliv** [reply]
Actually I do not like idea to move to another alternatives.

**Reply by MaceWindu** [reply]
Done

**Reply by viceroypenguin** [reply]
I've always preferred Discord to Slack. No reason we couldn't move to a Discord server. Bots are relatively easy to integrate with Azure.

**Reply by MaceWindu** [reply]
Anyway, if they will unblock us, we should transfer ownership to somebody else as @ili is not active for a long time. Otherwise we don't have any option except looking for alternative solution.

**Reply by sdanyliv** [reply]
@MaceWindu, maybe we have to do something in Nuget also? )

**Reply by MaceWindu** [reply]
We can. It was bothering me for a long time :)

**Reply by MaceWindu** [reply]
btw, same story happened with elixir lang workspace  :)

**Reply by igor-tkachev** [reply]
What is wrong with Nuget?

**Reply by MaceWindu** [reply]
nothing big, just removed users from list of owners for some old nugets, as they already own them as organisation members.

**Comment by MaceWindu** [comment]
Let's summarize it.

- workspance blocked without any notification
- there is no answer at all from their side after 3 emails sent

My conclusion: we should forget about slack as company with awful customer service and move on. E.g. to element

**Reply by viceroypenguin** [reply]
https://discord.gg/KYbqtjFgXw - Discord server; I'll give dev team access as they sign in.

NB: .net org and c# dev are also both on Discord

**Reply by sdanyliv** [reply]
@MaceWindu, let's try to connect to discord via link above (already here). Then we can discuss what to choose as a messenger if discord is not proper tool.

**Reply by jods4** [reply]
I would like to +1 Discord if it's ok for others as well.

I'm already on it for TS, Vite, Vue and more. 
It's convenient to have everything at one place, I connected to Slack only for linq2db, I'm more often on Discord.
